src/renderers/shaders/ShaderChunk/skinning_vertex.glsl

```diff
@@ -7,6 +7,7 @@
 	skinned += boneMatY * skinVertex * skinWeight.y;
 	skinned += boneMatZ * skinVertex * skinWeight.z;
 	skinned += boneMatW * skinVertex * skinWeight.w;
-	skinned  = bindMatrixInverse * skinned;
+
+	transformed = ( bindMatrixInverse * skinned ).xyz;
```

Comment:
You are right, but requiring the weights to be normalized actually follows from the original assumption that the matrices are affine. If the weights don't add up to one, then `skinned` would be incompatible with all affine transformations. In fact, such transformations need the w-component to be one if you intend to transform the vector correctly.

In any case, I explicitly checked the weights by logging them for a bunch of Three.js examples, and they all add up to one. I checked the matrices as well and the last row for each of them is [0, 0, 0, 1], making them affine. 

I believe in the end this is an entirely reasonable assumption to make.